synapse/models/risk.py

```diff
@@ -414,10 +414,10 @@ def getModelDefs(self):
 
                     ('verdict', ('risk:alert:verdict:taxonomy', {}), {
                         'ex': 'benign.false_positive',
-                        'doc': 'Analyst specified verdict taxonomy about why the alert is malicious or benign.'}),
+                        'doc': 'A user-specified verdict about why the alert is malicious or benign.'}),
```

Comment:
Should probably eliminate analyst/user specified here too. Could be machine specified as well.